invenio_accounts/tasks.py

```diff
@@ -30,7 +30,7 @@
 from flask import current_app
 
 
-@shared_task
+@shared_task(serializer='pickle')
```

Comment:
We shouldn't be using ``pickle`` as serialiser. It's insecure and will be disabled by default (http://docs.celeryproject.org/en/latest/whatsnew-3.1.html#last-version-to-enable-pickle-by-default).